### PR TITLE
Fixed webpack config file - took out empty string in extensions, adde…

### PR DESCRIPTION
**Pull request by Cole7787**
…d -loader to babel in loaders

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
 		]
 	}
 	, resolve: {
-		extensions: ['', '.js', '.jsx']
+		extensions: ['.js', '.jsx']
```
Comment (asBrettisay) [inline webpack.config.js:21]: This should not be removed. An empty string here allows extensions based on the extension specified when the file is required.

Here's the quote from the webpack documentation:

`If you want modules that were required with their extension (e.g. require('./somefile.ext')) to be properly resolved, you must include an empty string in your array. Similarly, if you want modules that were required without extensions (e.g. require('underscore')) to be resolved to files with ".js" extensions, you must include ".js" in your array.`

File: webpack.config.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ module.exports = {
 			{
 				test: /\.js/
 				, exclude: /node_modules/
-				, loader: 'babel'
+				, loader: 'babel-loader'
```
Comment (asBrettisay) [inline webpack.config.js:11]: [According to this](https://github.com/babel/babel-loader#the-node-api-for-babel-has-been-moved-to-babel-core) use of `babel` as as loader has been deprecated as of webpack 2.x. Nice catch!

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(
+  <div>
+
+  </div>,
```
Comment (asBrettisay) [inline src/index.js]: I like the idea of adding something basic here, what about adding a quick `<h1>Hello World!</h1>`?

**Review by asBrettisay — CHANGES_REQUESTED** [review]
This looks good but need some changes.

Also need to `.gitignore` your `bundle.js` file, it was included in the PR but we don't want it.

File: src/index.js (hunk 3 above)
Comment (r-walsh) [reply src/index.js]: This was set up for starting React app's from scratch. I'd say we should either transition it to CRA or leave it a blank slate. Any middle ground doesn't seem beneficial to me with the upcoming curriculum changes.

**Pull request closed without merging**